scripts/add.tokens.erc20.mjs

```diff
@@ -0,0 +1,264 @@
+import { load } from 'cheerio';
+import dotenv from 'dotenv';
+import { ethers } from 'ethers';
+import fs from 'fs/promises';
+import fetch from 'node-fetch';
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import path from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const rootPath = resolve(__dirname, '../');
+
+const envFile = process.env.NODE_ENV === 'production' ? '.env' : '.env.development';
+const envPath = resolve(rootPath, envFile);
+dotenv.config({ path: envPath });
+
+const getArgValue = (argName) => {
+	const argIndex = process.argv.indexOf(argName);
+	return argIndex > -1 && argIndex < process.argv.length - 1 ? process.argv[argIndex + 1] : null;
+};
+
+const ETHERSCAN_API_KEY = getArgValue('--etherscan-api-key') || process.env.VITE_ETHERSCAN_API_KEY;
```

Comment:
prefer nullish coalescing instead of `||` which is less prone to unwanted errors.

ps: we should set an eslint rule for that or did we already?

Reply:
changed!

so, we have that rule set, but the `scripts` folder was not included in `tsconfig.eslint.json`

```json
"include": [
		"ambient.d.ts",
		"non-ambient.d.ts",
		"./types/**/$types.d.ts",
		"./vite.config.js",
		"./vite.config.ts",
		"./src/**/*.js",
		"./src/**/*.ts",
		"./src/**/*.svelte",
		"./vitest.setup.ts",
		"./vitest.config.ts",
		"./tailwind.config.js",
		"./svelte.config.js",
		"./postcss.config.js",
		"./playwright.config.ts",
		"./e2e/**/*.js",
		"./e2e/**/*.ts",
		"./scripts/*.mjs"  <-- to be included
	],
```

i can do it in another PR; let's note that we need to create the exception that for those specific files we accept `console.log`